fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticConstants.java

```diff
@@ -79,7 +79,7 @@ public class StatisticConstants {
     public static final int LOAD_RETRY_TIMES = 3;
 
     // union more relation than 512 may cause StackOverFlowException in the future.
```

Comment:
is there overflow? UNION_ALL_LIMIT is more than 512

Reply:
No, in fact we could union even ten thousand  query according to my test on legacy optimizer